### PR TITLE
DSG for localize pages and release-notes

### PR DESCRIPTION
**Pull request by josephgregoryii**
## Related
https://github.com/newrelic/gatsby-theme-newrelic/pull/623

### CODE REVIEW

**Comment by josephgregoryii** [comment]
This is still a WIP. PR created for preview

File: gatsby-node.js [hunk 1]
```diff
@@ -265,11 +265,15 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
...
-        disableSwiftype: !i18nNode,
-      });
+      createPageFromNode(
+        i18nNode || node,
+        {
+          prefix: i18nNode ? '' : locale,
+          createPage,
+          disableSwiftype: !i18nNode,
+        },
+        !!i18nNode // defer if localized
```
Comment (roadlittledawn) [inline gatsby-node.js:275]: i think we'd always want to defer these (localized pages w/english and localized pages with translated content). the check if it's an `i18nNode` in `disableSwiftype` is to ensure we're only indexing pages with translated content in the search engine.

Comment (josephgregoryii) [reply gatsby-node.js:275]: This makes sense and clarifies the logic here then. I can fix that 👍🏻

File: gatsby-node.js [hunk 2]
```diff
@@ -426,6 +431,7 @@ const createPageFromNode = (
         fileRelativePath,
         layout: 'basic',
       },
+      defer: defer,
```
Comment (roadlittledawn) [inline gatsby-node.js:434]: i don't think we need to defer these. this is just a way to show a missing template page in local development in the case we can't find a template for a given MDX file type.

**Comment by josephgregoryii** [comment]
Converted to draft to allow discussion around https://github.com/newrelic/gatsby-theme-newrelic/issues/620

File: gatsby-node.js [hunk 3]
```diff
@@ -416,6 +421,7 @@ const createPageFromNode = (
         slugRegex: `${slug}/.+/`,
         disableSwiftype,
       },
+      defer: defer,
```
Comment (rudouglas) [inline gatsby-node.js:424]: Could change to `defer,`

**Comment by aswanson-nr** [comment]
Looks like most of the links on the homepage aren't working in the preview!

**Comment by rudouglas** [comment]
This still needs the theme bump after merging https://github.com/newrelic/gatsby-theme-newrelic/issues/620

**Comment by josephgregoryii** [comment]
Lets wait on merging for now to allow discussion around the fix

**Review by roadlittledawn — APPROVED** [review]
:shipit:

**Pull request merged**